src/services/userService.js

```diff
@@ -1,8 +1,12 @@
 import api from 'api';
 
 class UserService {
-  static login(user) {
-    return api.post('/users/sign_in', user);
+  static login() {
```

Comment:
Don't you have a login endpoint? :thinking: Maybe add a TODO comment if this is a temporary change

Reply:
Auth is not implemented yet, I will add a todo comment, good point